pkg/scripts/download_test.go

```diff
@@ -30,8 +30,8 @@ func (f *FakeHTTPGet) get(url string) (*http.Response, error) {
 	}, f.err
 }
 
-func getHTTPReader() (*HttpURLReader, *FakeHTTPGet) {
-	sr := &HttpURLReader{}
+func getHTTPReader() (*HTTPURLReader, *FakeHTTPGet) {
```

Comment:
it's not that important (so i'm not asking you do it, it's fine as is) but it does bother me a little that they are out of sync now.  it may be more critical to have renamed the constructor, but that doesn't make it unimportant to align other stuff.